src/components/modules/MainPage/MainCom/MainCom.js

```diff
@@ -0,0 +1,65 @@
+import { Layout,ProjectContainer } from "./StyledMain";
+import LongCard from "../src/component/LongCard";
+import CARDDATA from "../src/CardData";
+import {connect} from 'react-redux';
+import React from "react";
+import {withRouter} from "react-router-dom";
```

Comment:
routing is handled using next.js so this line can be removed

Reply:
deleted and updated it ! thank you !